docs/content/preview/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -170,22 +178,86 @@ If a database or a keyspace has an associated snapshot schedule, you can use tha
                                              minus 1h
     ```
 
-    For detailed information on the relative time formatting, refer to the [`restore_snapshot_schedule` reference](../../../admin/yb-admin/#restore-snapshot-schedule).
+For detailed information on the relative time formatting, refer to the [`restore_snapshot_schedule` reference](../../../admin/yb-admin/#restore-snapshot-schedule).
```

Comment:
Keep it indented; it's part of the "Restore to a time that is relative..." bullet.
```suggestion
    For detailed information on the relative time formatting, refer to the [`restore_snapshot_schedule` reference](../../../admin/yb-admin/#restore-snapshot-schedule).
```